scr/main/java/bittercode/model/Cart.java

```diff
@@ -0,0 +1,31 @@
+package com.bittercode.model;
+
+import java.io.Serializable;
+
+public class Cart implements Serializable {
+
+    private Book book;
+    private int quantity;
+
+    private Cart(Book book, quantity) {
+        this.book = book;
+        this.quantity = quantity;
+    }
+
+    public Book getBook() {
+        return book;
+    }
+
+    public void setBook(Book book) {
+        this.book = book;
+    }
+
+    public int getQuantity() {
+        return Quantity;
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Typo in getQuantity method return statement</b></div>
<div id="fix">

There's a typo in the `getQuantity` method. `Quantity` should be lowercase `quantity` to match the field name.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -24,1 +24,1 @@
 -        return Quantity;
 +        return quantity;
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/PrajaktaBendreBito/BookStore/pull/4#issuecomment-2434904657>#809440</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged